controllers/recruitersController.js

```diff
@@ -13,7 +13,7 @@ const addRecruiter = async (req, res) => {
     return res.json(result)
   } catch (error) {
     logger.error(`Error while adding recruiterInfo: ${error}`)
-    return res.boom.serverUnavailable('Something went wrong please contact admin')
+    return res.boom.notFound('Something went wrong please contact admin')
```

Comment:
`notFound`? 🤔

Reply:
Should I change it back to serverUnavailable?

Reply:
Yes. If you want to handle the case of concerned user not found, that has to be handled differently. Please check `makeNewBid` function in the `auctionController.js` to get a fair idea of how to handle such things. Also see the concerned model function.